src/validation/rules.rs

```diff
@@ -100,6 +103,9 @@ impl Rule for RuleSet {
 }
 
 /// Get default rules for a given `buffrs` package name.
-pub fn package_rules(name: &str) -> RuleSet {
-    vec![Box::new(PackageName::new(name)), Box::new(IdentCasing)]
+pub fn all(manifest: &PackageManifest) -> RuleSet {
```

Comment:
I love this change, because it means that we could at some point allow you to configure rules in the `Proto.toml` file, similar to how cargo lets you do it now:

```toml
[lints.rustc]
missing_docs = "warn"
```